products/tarantool.md

```diff
@@ -0,0 +1,70 @@
+---
+permalink: /tarantool
+layout: post
+title: Tarantool
+command: $ tarantool --version
+releasePolicyLink: https://www.tarantool.io/en/doc/latest/release/policy/
+changelogTemplate: |
+  https://www.tarantool.io/en/doc/latest/release/__LATEST__/
```

Comment:
Sadly, the release notes for 1.10.12 and 2.8.3 are still not on the website. However they're available on GitHub. At least now it would be better to point the links to GitHub.

Reply:
Done

Reply:
But this template does not work for other releases too. Say, for 2.8.2. While GitHub releases have perfectly preditable URLs.

Reply:
* 1.10.2: https://www.tarantool.io/en/doc/latest/release/1.10/#whats-new-1102
* 1.10.10: https://www.tarantool.io/en/doc/latest/release/1.10.10/
* 1.10.11: https://www.tarantool.io/en/doc/latest/release/2021-08-releases/

Reply:
updated tempate:

```diff
--- a/products/tarantool.md                                                 
+++ b/products/tarantool.md
@@ -6,7 +6,7 @@ command: $ tarantool --version              
 releaseImage: https://www.tarantool.io/en/doc/latest/_images/releases_calendar.svg
 releasePolicyLink: https://www.tarantool.io/en/doc/latest/release/policy/
 changelogTemplate: |                                                       
-  https://www.tarantool.io/en/doc/latest/release/__LATEST__/
+  https://github.com/tarantool/tarantool/releases/tag/__LATEST__
 category: db                      
 iconSlug: NA                                                               
 eolColumn: Support Status  
```